python/ray/train/v2/torch/train_loop_utils.py

```diff
@@ -38,11 +39,119 @@
 )
 
 
+@PublicAPI(stability="stable")
+@requires_train_worker()
 def get_device() -> torch.device:
+    """Gets the correct torch device configured for the current worker.
+
+    Returns the torch device for the current worker. If more than 1 GPU is
+    requested per worker, returns the device with the lowest device index.
+
+    .. note::
+
+        If you requested multiple GPUs per worker, and want to get
+        the full list of torch devices, please use
+        :meth:`~ray.train.torch.get_devices`.
+
+    Assumes that `CUDA_VISIBLE_DEVICES` is set and is a
+    superset of the `ray.get_gpu_ids()`.
+
+    Returns:
+        The torch device assigned to the current worker.
+
+    Examples:
+
+        Example: Launched 2 workers on the current node, each with 1 GPU
+
+        .. testcode::
+            :skipif: True
+
+            os.environ["CUDA_VISIBLE_DEVICES"] = "2,3"
+            ray.get_gpu_ids() == [2]
+            torch.cuda.is_available() == True
+            get_device() == torch.device("cuda:0")
+
+        Example: Launched 4 workers on the current node, each with 1 GPU
+
+        .. testcode::
+            :skipif: True
+
+            os.environ["CUDA_VISIBLE_DEVICES"] = "0,1,2,3"
+            ray.get_gpu_ids() == [2]
+            torch.cuda.is_available() == True
+            get_device() == torch.device("cuda:2")
+
+        Example: Launched 2 workers on the current node, each with 2 GPUs
+
+        .. testcode::
+            :skipif: True
+
+            os.environ["CUDA_VISIBLE_DEVICES"] = "0,1,2,3"
+            ray.get_gpu_ids() == [2,3]
+            torch.cuda.is_available() == True
+            get_device() == torch.device("cuda:2")
+
+
+        You can move a model to device by:
+
+        .. testcode::
+            :skipif: True
+
+            model.to(ray.train.torch.get_device())
+
+        Instead of manually checking the device type:
+
+        .. testcode::
+            :skipif: True
+
+            model.to("cuda" if torch.cuda.is_available() else "cpu")
+    """
     return get_devices()[0]
 
 
+@PublicAPI(stability="beta")
+@requires_train_worker()
 def get_devices() -> List[torch.device]:
+    """Gets the correct torch device list configured for the current worker.
```

Comment:
```suggestion
    """Gets the list of torch devices configured for the current worker.
```

The original wording implies that there are multiple torch device lists for the current worker, some of which are wrong.